inc/domainrecord.class.php

```diff
@@ -339,6 +260,29 @@ function showForm($ID, $options = []) {
       echo "</td>";
       echo "</tr>";
 
+      echo "<tr class='tab_bg_1'>";
+      echo "<td>" . __('Data') . "</td>";
+      echo "<td colspan='3'>";
+      Html::autocompletionTextField($this, "data");
```

Comment:
Some records can have lots of chars in their data (`TXT` records for example). Maybe having a textarea here could be more readable (but we will loose the auutocomplete feature).
By the way, autocomplete needs to be declared in the corresponding SO.

Reply:
OK, I've enabled autocompletion for now, we'll see what is the usage , this is quite easy to change